pkg/resources/create_resource.go

```diff
@@ -0,0 +1,25 @@
+package resources
```

Comment:
Suggestion: we really don't need separate package for this functionality rather we can put this `CreateFromYAML()` in this `toolchain.go` file [link](https://github.com/codeready-toolchain/toolchain-operator/blob/master/pkg/toolchain/toolchain.go).  `toolchain.go` file holds common functionality across resource/operators.

Reply:
I don't lile the idea of having a single file (here `toolchain.go`) which mixes unrelated funcs. I'd rather move the `create_resource.go` file somewhere else if you don't like the idea of having a separated pkg, but keep the code in its own file.

Reply:
@xcoulon Yes, I guess moving `create_resource.go` to `toolchain` pkg is one option .. and if possible, rename file to `resource.go`.  Now, I will left everything up to you :+1:

Reply:
please see my comment in https://github.com/codeready-toolchain/toolchain-operator/pull/16#discussion_r350600268